hazelcast/src/test/java/com/hazelcast/ringbuffer/impl/RingbufferAbstractTest.java

```diff
@@ -710,7 +711,7 @@ public void readManyAsync_whenMoreAvailableThanMaximum() throws ExecutionExcepti
         ReadResultSet<String> resultSet = f.get();
 
         assertNotNull(resultSet);
-        assertEquals(asList("1", "2"), resultSet);
+        assertThat(f.get(), contains("1", "2"));
```

Comment:
from the contract of `contains()`
```
   * Creates a matcher for {@link Iterable}s that matches when a single pass over the
   * examined {@link Iterable} yields a series of items, each logically equal to the
   * corresponding item in the specified items.  For a positive match, the examined iterable
   * must be of the same length as the number of specified items.
```

so it will match if the length is same with the same items in the same order. that's exactly what we want, isn't it?